rpcs3/rpcs3qt/settings_dialog.h

```diff
@@ -60,6 +60,9 @@ private Q_SLOTS:
 	bool m_use_discord;
 	QString m_discord_state;
 
+	// Lib loading state
+	int last_lib_mode = -1;
```

Comment:
Ah, now I understand. You meant the reset of unapplied values when clicking the same button after changing the libs.
Anyway, here some advise:
```suggestion
	int m_last_lib_mode = -1;
```

Reply:
Fixed.